### PR TITLE
LPS-172219 Fix POSHI automation | Master

### PR DESCRIPTION
**Pull request by alessandromalves**
This is only for master
https://issues.liferay.com/browse/LPS-172219
cc.: @david-gutierrez-mesa @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by alessandromalves** [comment]
ci:test:sf

**Comment by alessandromalves** [comment]
Hey @Tim-Cao, I also updated the two tests in addition to updating the paths. I removed the `AssertVisible` function from the tests because I saw that an assert was performed inside `Uncheck.uncheckNotVisible`. I also added `Staging.publishToLive();` in place of the other functions.

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/staging/StagingPublishToLive.path [hunk 1]
```diff
@@ -123,12 +123,12 @@
...
 <tr>
 	<td>PAGES_TO_PUBLISH_CHECKBOX_CHECKED</td>
-	<td>//div[@class='pages-selector']//div[normalize-space()='${key_entryName}' and contains(@class,'checked')]//span[contains(@class,'icon-check')]</td>
+	<td>//span[normalize-space(text())='${key_entryName}']//ancestor::div[@class='autofit-row']//input[@type='checkbox' and (@checked)]</td>
 	<td></td>
 </tr>
 <tr>
 	<td>PAGES_TO_PUBLISH_CHECKBOX_UNCHECKED</td>
-	<td>//div[@class='pages-selector']//div[normalize-space()='${key_entryName}' and not(contains(@class,'checked'))]//span[contains(@class,'icon-check')]</td>
+	<td>//span[normalize-space(text())='${key_entryName}']//ancestor::div[@class='autofit-row']//input[@type='checkbox' and not(@checked)]</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/staging/StagingPublishToLive.path:131]: `//div[contains(@class,'layout-selector')]//div[contains(@class,'treeview-link')][*//div[normalize-space()='${key_entryName}']]//input[@type='checkbox'][not(@checked)]`

**Review by Tim-Cao — COMMENTED** [review]
Hi @alessandromalves, would you mind modifying according to comments? Thanks

**Comment by alessandromalves** [comment]
ci:test:sf

**Comment by alessandromalves** [comment]
ci:forward

**Pull request closed without merging**